honeybee_radiance/cli/octree.py

```diff
@@ -73,7 +73,10 @@ def create_octree_from_folder(
     model_folder = ModelFolder.from_model_folder(folder)
     try:
         black_out = False if default else True
-        scene_files = model_folder.scene_files(black_out=black_out)
+        try:
+            scene_files = model_folder.scene_files(black_out=black_out)
```

Comment:
@mikkelkp, we probably want to add this check to the honeybee-radiance-folder library, and fix the issue in the source.